tests/regression/regression.py

```diff
@@ -24,16 +25,64 @@
 #
 
 
-from pyBeam import CBeamSolver
+import pdb
```

Comment:
why is this pdb instead of pyBeam?

Reply:
Ok, this was a reorganisation of the code. pdb was just for debug. All the swigged classes are inside the folder swig. import swig does all the work